srcpkgs/epson-inkjet-printer-escpr/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'epson-inkjet-printer-escpr'
 pkgname=epson-inkjet-printer-escpr
-version=1.7.22
+version=1.7.25
 revision=1
 build_style=gnu-configure
 configure_args="--disable-static --with-cupsppddir=/usr/share/ppd --with-cupsfilterdir=/usr/lib/cups/filter"
 makedepends="cups-devel"
 depends="cups-filters"
 short_desc="Epson Inkjet Printer Driver (ESC/P-R)"
-maintainer="Justin Berthault <justin.berthault@zaclys.net>"
+maintainer="Mohammed Anas <triallax@tutanota.com>"
 license="GPL-2.0-or-later"
-homepage="http://download.ebz.epson.net/dsc/search/01/search/?OSC=LX"
-distfiles="https://download3.ebz.epson.net/dsc/f/03/00/13/96/55/c6fced63098ae1ba104f11f572794fd558ffca29/${pkgname}-${version}-1lsb3.2.tar.gz"
-checksum=0b2fefa7fd077b6fdf778cb53a8c60bf999a396ef88145182c5c4a08f34c652c
+homepage="https://download.ebz.epson.net/dsc/search/01/search/?OSC=LX&productName=B700"
```

Comment:
Unlike the old homepage, this one will actually show the driver in the search results at least.